components/docs-chef-io/content/automate/ha_healthcheck.md

```diff
@@ -21,78 +21,141 @@ This page includes commands that can be executed for the Chef Automate cluster p
 ## Automate HA Service Commands
 
 - Get the Automate HA cluster Information 
-  ```cmd
+  
+```cmd
     chef-automate info 
-  ```
+```
 
 - Post Deployment, run smoke test cases on Automate HA cluster, please run the command from bation node
```

Comment:
Post Deployment, run the smoke test cases on Automate HA cluster and run the command from the bastion node.